internal/serve/httphandler/payments_handler.go

```diff
@@ -176,6 +180,7 @@ func (p PaymentsHandler) producePaymentsReadyEvents(ctx context.Context, payment
 		paymentsReadyToPay.Payments = append(paymentsReadyToPay.Payments, schemas.PaymentReadyToPay{ID: payment.ID})
 	}
 	msg.Data = paymentsReadyToPay
+	msg.Key = paymentsReadyToPay.TenantID
```

Comment:
`Key` in Kafka is useful especially for partitioning. Messages with the same key are handled by the same partition. Which also means that messages with the same Key will also be handled in order. 

In the case of retry payments, given that payments can be part of different disbursements, the only key we can use is the tenant ID.